project_proposals/RCM OSS.md

```diff
@@ -0,0 +1,127 @@
+# Questionnaire for RCM OSS intending to join the OpenRail Association
+### What is the project's name?
+RCM OSS - Rail Condition Monitoring Open Source Software
+##
+### Describe the project. What does the project do, why is it valuable, where does it come from?
+RCM by SBB comprises a suite of products for rail condition monitoring. Based on the open source data format RCM-DX (RCM Data eXchange), RCM enables easy accessibility and visualisation of railway diagnostic data.
+The aim of RCM OSS is to make rail condition data easy to access, store and visualize. Therefore, RCM OSS is beneficial to railway companies as well as measurement system suppliers.
+##
+### Who are the maintainers of the project (these will be the primary contacts for the OpenRail Association)?
+* Stephanie Schalbetter: stephanie.schalbetter@sbb.ch
+* Jean-Frédéric Bonjour: jean-frederic.bonjour@sbb.ch
+##
+### Which organizations are sponsoring/contributing to the project?
+Swiss Federal Railways SBB
+##
+### Where is the code hosted?
+* [Viewer (Freeware)](https://github.com/SchweizerischeBundesbahnen/rcm-dx/releases)
+* [RCM DX Format Specification](https://github.com/SchweizerischeBundesbahnen/rcm-dx)
+* [Example Data Track (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TGMS_v003_created-20220429-065432.rcmdx)
+* [Example Data Position (100 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TLS_v001_created-20220429-065432.rcmdx)
+* [Example Data Track Inspection (167 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TSIMS-RAW-COMP_v001_created-20220429-065432.rcmdx)
+* [Example Data Video (937 MB)](https://sbb-wcms-big-files-server-prod.s3.eu-central-1.amazonaws.com/RCM-DX_Files/20220428-114946_DFZ00_TVS_v001_created-20220429-065432.rcmdx)
+##
+### What is the project's main license?
+Eclipse Public License 2.0
+##
+### What other licenses does the project use, e.g. for included 3rd party code?
+Code is not Open Source:
```

Comment:
We have not yet published any source code and only the specification of the format has been published. Would it be enough to show the dependencies as soon as we release code? Either way, the distinction you have made makes sense.

Reply:
I think it would be ok to handle the question of dependencies when code is released. It has to be done, before the repository can be added to the OpenRail Association organization. This could be achieved in two ways, either it is published somewhere else already, we do the license review there, and then move it to the OpenRail Association organization. Or it is started as a private repository, we do the license review in preparation of publishing the repository, and when this is done and licenses are clear we publish it in the OpenRail Association organization.

It would be good to make it clear in the questionnaire what the current and future repositories will be. It's ok if not all of them are ready to be published and we can work on this as part of the incubation process after initial acceptance of the project. But it would be good to know exactly what the scope of the proposal is.

One alternative might be to handle this as two proposals, one for the specification, and one for the code. But as these belong together it would also make sense to group them in one proposal. If possible, it should be clear what path we go from the beginning, though.

Reply:
We like to handel this as one proposal. Only the content in the public repository [RCM DX Format Specification](https://github.com/SchweizerischeBundesbahnen/rcm-dx) should be transferred. This includes the viewer as a binary, which can be used as freeware in the public repository. Is this a problem? The dependencies all refer to the viewer whose code we currently do not want to publish. Later on, the new open-scource developed code of the RCM-DX R/W library will be added into the same repository. The closed-source code of the viewer stays in a privat SBB repository. We do not consider it realistic to publish the code of the viewer as a whole in the next 2 years. The effort required for this exceeds our current capabilities. I hope this is now clear in the new version of the questionnaire.